src/modules/evaluateExpression/evaluateExpression.ts

```diff
@@ -1,46 +1,11 @@
-interface IParameters {
-  [key: string]: any
-  connection?: IConnection
-}
-
-interface IQueryNode {
-  value?: string | number | boolean | object
-  type?: NodeType
-  operator?: Operator
-  children?: Array<IQueryNode>
-}
-
-interface IConnection {
-  user?: string
-  host?: string
-  database?: string
-  password?: string
-  port?: number
-  query?: any
-}
-
-type NodeType = 'string' | 'number' | 'boolean' | 'array'
-
-type Operator =
-  | 'AND'
-  | 'OR'
-  | 'CONCAT'
-  | '='
-  | '!= '
-  | '+'
-  | 'REGEX'
-  | 'objectProperties'
-  | 'pgSQL'
-  | 'graphQL'
-
-const defaultParameters: IParameters = {
-  connection: {},
-}
+import { IConnection, IQueryNode, IParameters } from './types'
 
 export default async function evaluateExpression(
   inputQuery: IQueryNode | string,
-  params = defaultParameters
+  params: IParameters
```

Comment:
Ah, this seems to have broken the `evaluateExpression` test suite. Gets the error ` Expected 2 arguments, but got 1.` for all tests. I think the `default_parameters` was supplying an empty connection object, which is now missing.

I suggest don't worry about this for now. I'm going to re-write the input parameter structure anyway, when I revisit the `objectProperties` operator, so I can fix it with that.

Reply:
Ok. Sorry about that. thanks

Reply:
Oh, I'm actually returning this since the tests aren't working. I'll add some commits.